doc/release/yarp_3_4/detect_clock_reset.md

```diff
@@ -0,0 +1,8 @@
+detect_clock_reset {#yarp_3_4}
+-----------------------
+
+### Libraries
+
+#### network clock
+
+* unblock all waiting threads if a network reset is detected
```

Comment:
Documentation follows the general rules of English, so sentence should start with a capital letter and end with a point. Furthermore, it would be good to provide a bit more context for the reader, that otherwise could ask "What is a "network reset"?" The threads that are waiting on what? 

```suggestion
* If in a `yarp::os::NetworkClock` a clock reset is detected, unblock all the threads that are blocked waiting on a  `yarp::os::NetworkClock::delay` call on that network clock. A network clock reset is defined a jump in the past of the time published by the network clock port. This avoid that if `YARP_CLOCK` is set,  `yarp::os::PeriodicThread` silently stops to run if a network clock is reset. 
```